### DISCUSSION TITLE
[2022/02/16] KubeSphere community meeting

### DISCUSSION THREAD

**Discussion by zryfish** [kubesphere/kubesphere#4673] [Community Meeting]
Topic: KubeSphere v3.3 Project
Location: Online
Time: 2022/02/16 16:00 PM Beijing Time
Tencent Meeting: 643-8877-6376

会议主题：KubeSphere Community Meeting
会议时间：2022/02/16 16:00-17:00 (GMT+08:00) 中国标准时间 - 北京
点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/HOlnPLhwtQLu
#腾讯会议：643-8877-6376

**Comment by JohnNiang** [comment]
Meeting Record: https://www.bilibili.com/video/BV1dq4y147j8
